.coveragerc

```diff
@@ -242,6 +242,7 @@ omit =
     homeassistant/components/climate/heatmiser.py
     homeassistant/components/climate/homematic.py
     homeassistant/components/climate/knx.py
+    homeassistant/components/climate/mqtt.py
```

Comment:
thanks for the hint! i was copying most of fan/mqtt, and the PR template checkbox `New files were added to .coveragerc` wasn't particularly helpful when the `omit =` is far away from the new line added.